balancer-js/examples/joinGeneralisedComposableStable.ts

```diff
@@ -5,36 +5,38 @@
  *   anvil --fork-url https://rpc.ankr.com/eth --fork-block-number 16411000 --fork-chain-id 1
```

Comment:
Maybe we should consolidate to one example as `joinGeneralised` seems very similar?